src/ui/popup.js

```diff
@@ -231,6 +234,22 @@ export default class Popup extends Evented {
         return this.setDOMContent(frag);
     }
 
+    /**
+     * Sets the popup's max width.
+     *
+     * @param pixel A string representing the pixel value for the maximum width.
+     * @returns {Popup} `this`
+     */
+    getMaxWidth() {
+      return this._container;
+    }
+
+    setMaxWidth(maxWidth: string) { // should this be number?
+      this.options.maxWidth = maxWidth;
+      this._update();
+      return this;
```

Comment:
If you convert this to a setter method, there's no reason to `return this` at the end (because it won't be a chainable method anymore).